views/components/form-fields-next/macros.njk

```diff
@@ -351,7 +349,7 @@
         <div class="o-details__content ff-address__fields">
             <div class="ff-address__field">
                 {{ inputText({
-                    label: 'Building and street',
+                    label: copy.fields.address.fieldNames.line1,
```

Comment:
It will be interesting to see how this feels when we build subsequent forms, i.e. it might be better to consider these bits of copy "global" with explicit `__('commonFields.address...')` style calls rather than implicit `copy` object. This is good for now though.